resources/public/api/conf/1.0/application.yaml

```diff
@@ -204,22 +195,104 @@ paths:
-                - employerPayeReference: 123/AB456
-                  taxTakenOffPay: 890.35
-                - employerPayeReference: 456/AB456
-                  taxTakenOffPay: 224.99
+                  - $ref: '#/components/schemas/individualPayeTestSupportSaBenefitsAnnualSummaryResponse'
+              examples:
+                happy-path-1:
+                  $ref: '#/components/examples/taxResponseHappyPath1'
+                happy-path-2:
+                  $ref: '#/components/examples/taxResponseHappyPath2'
       deprecated: false
       security:
-        - application-restricted: []
+        - application-restricted: [ ]
 components:
+  examples:
+    benefitsRequestHappyPath1:
+      description: Payload for scenario HAPPY_PATH_1
+      value:
+        scenario: HAPPY_PATH_1
+    benefitsRequestHappyPath2:
+      description: Payload for scenario HAPPY_PATH_2
+      value:
+        scenario: HAPPY_PATH_2
+    benefitsResponseHappyPath1:
+      description: Response for scenario HAPPY_PATH_1 with all fields present
+      value:
+        employments:
+          - employerPayeReference: 123/AB456
+            companyCarsAndVansBenefit: 44
+            fuelForCompanyCarsAndVansBenefit: 99
+            privateMedicalDentalInsurance: 64
+            vouchersCreditCardsExcessMileageAllowance: 80
+            goodsEtcProvidedByEmployer: 125
+            accommodationProvidedByEmployer: 375
+            otherBenefits: 16
+            expensesPaymentsReceived: 415
+          - employerPayeReference: 456/AB456
+            companyCarsAndVansBenefit: 50
+            fuelForCompanyCarsAndVansBenefit: 249
+            privateMedicalDentalInsurance: 75
+            vouchersCreditCardsExcessMileageAllowance: 23
+            goodsEtcProvidedByEmployer: 250
+            accommodationProvidedByEmployer: 275
+            otherBenefits: 87
+            expensesPaymentsReceived: 265
+    benefitsResponseHappyPath2:
+      description: Response for scenario HAPPY_PATH_2 with only mandatory fields present
+      value:
+        employments:
+          - employerPayeReference: 123/AB456
+    employmentsRequestHappyPath1:
+      description: Payload for scenario HAPPY_PATH_1
+      value:
+        scenario: HAPPY_PATH_1
+    employmentsRequestHappyPath2:
+      description: Payload for scenario HAPPY_PATH_2
+      value:
+        scenario: HAPPY_PATH_2
+    employmentsResponseHappyPath1:
+      description: Response for scenario HAPPY_PATH_1 with all fields present
+      value:
+        employments:
+          - employerPayeReference: 123/AB456
+            employerName: Company XYZ
+            offPayrollWorkFlag: true
+          - employerPayeReference: 456/AB456
+            employerName: Company ABC
+            offPayrollWorkFlag: false
+    employmentsResponseHappyPath2:
+      description: Response for scenario HAPPY_PATH_2 with only mandatory fields present
+      value:
+        employments:
+          - employerPayeReference: 123/AB456
```

Comment:
`employerName` is mandatory and has been added to [HAPPY_PATH_2](https://github.com/hmrc/paye-des-stub/blob/DDCE-1709/resources/public/scenarios/individual-employment/HAPPY_PATH_2.json) correctly. Please include it in this example, thanks.